--- a/net/minecraft/server/PacketPlayOutWorldBorder.java
+++ b/net/minecraft/server/PacketPlayOutWorldBorder.java
@@ -18,8 +18,10 @@
 
     public PacketPlayOutWorldBorder(WorldBorder worldborder, PacketPlayOutWorldBorder.EnumWorldBorderAction packetplayoutworldborder_enumworldborderaction) {
         this.a = packetplayoutworldborder_enumworldborderaction;
-        this.c = worldborder.getCenterX();
-        this.d = worldborder.getCenterZ();
+        // CraftBukkit start - multiply out nether border
+        this.c = worldborder.getCenterX() * (worldborder.world.worldProvider instanceof WorldProviderHell ? 8 : 1);
+        this.d = worldborder.getCenterZ() * (worldborder.world.worldProvider instanceof WorldProviderHell ? 8 : 1);
+        // CraftBukkit end
         this.f = worldborder.getSize();
         this.e = worldborder.j();
         this.g = worldborder.i();
@@ -110,10 +112,6 @@
         packetlistenerplayout.a(this);
     }
 
-    public void a(PacketListener packetlistener) {
-        this.a((PacketListenerPlayOut) packetlistener);
-    }
-
     public static enum EnumWorldBorderAction {
 
         SET_SIZE, LERP_SIZE, SET_CENTER, INITIALIZE, SET_WARNING_TIME, SET_WARNING_BLOCKS;
